UnitedIncome/components
# [APP-5042] Convert MessageCard to TS

### Description
> Please provide a brief description of your changes below.

Converts the `MessageCard` component to use TypeScript. Also moved the `SimpleSummary` tests to be a `tsx` file as I missed that in the last PR.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Observe the typescriptness in all its glory

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

N/A

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-5042)
